.github/workflows/test_branches.yml

```diff
@@ -145,7 +145,7 @@ jobs:
     #    key: pkg-${{env.CACHE_VER}}.0-${{runner.os}}
 
     - name: TPL package download cache
-      uses: actions/cache@v2
+      uses: actions/cache@v3
```

Comment:
W should probably also update the action version for the commented out sections above (and in the next job below)